website/docs/google-docs-explainer.md

```diff
@@ -41,6 +41,13 @@ applies (? - needs verification) to the text seen _since_ the previous paragraph
 
 Paragraph styles set on a character level are either `null` or an object with the following properties:
 
+`ps_al`: `integer` between `0` and `3`
+
+- `0`: Left align text
```

Comment:
I thought these values indicate "start"/"end" alignments, not "left"/"right", no? Otherwise we'd have to throw errors for rtl directions in the google docs paste converter as well.

Reply:
You're correct! I forgot to update this 🙃